src/test/java/seedu/edulog/logic/parser/RevenueCommandParserTest.java

```diff
@@ -1,26 +1,32 @@
 package seedu.edulog.logic.parser;
 
+import static seedu.edulog.logic.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
 import static seedu.edulog.logic.parser.CommandParserTestUtil.assertParseFailure;
 import static seedu.edulog.logic.parser.CommandParserTestUtil.assertParseSuccess;
 
 import org.junit.jupiter.api.Test;
 
+import seedu.edulog.logic.commands.PaidRevenueCommand;
 import seedu.edulog.logic.commands.RevenueCommand;
+import seedu.edulog.logic.commands.UnpaidRevenueCommand;
 
 
 public class RevenueCommandParserTest {
     private RevenueCommandParser parser = new RevenueCommandParser();
 
     @Test
     public void parse_validArgs_successful() {
-        assertParseSuccess(parser, "paid", new RevenueCommand("paid"));
-        assertParseSuccess(parser, "unpaid", new RevenueCommand("unpaid"));
+        assertParseSuccess(parser, "paid", new PaidRevenueCommand());
+        assertParseSuccess(parser, "unpaid", new UnpaidRevenueCommand());
     }
 
     @Test
     public void parse_invalidArgs() {
```

Comment:
it would be great if you could add in comments so that others know what equivalence partition these test cases belong to